### ISSUE TITLE
Improvements to garner.a_vs_an

### ISSUE THREAD

**Issue by suchow** [amperser/proselint#79]
- [ ] Handle acronyms (e.g., "a FBI investigation"), but take into account acronyms that are pronounced as words instead of as their component letters.
- [ ] Better handle words that aren't in the pronunciation dictionary. Right now, they are never flagged (e.g., "an European country").

**Comment by Decagon** [comment]
I wasn't able to find the `garner.a_vs_an` rule anywhere. Has it been renamed?

**Comment by suchow** [comment]
No, it was temporarily removed because it depended on the natural language processing toolkit, a dependency that we didn't want to keep. Reintroducing it will require having access to the CMU pronunciation dictionary, which is doable but will take a little work unless there is a Python package already available.
